components/RunsPage/RunsTable/RunsTable.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import type { CategoryViewModel } from '~/lib/api/data-contracts'
+
+interface RunsTableProps {
+  category: CategoryViewModel
+}
+
+defineProps<RunsTableProps>()
+
+// TODO: Fetch runs
+</script>
+
+<template>
+  <table class="table h-fit flex-1 table-auto">
+    <thead class="table-header-group bg-bg-table-row">
+      <tr class="text-left">
+        <th class="pb-3 pl-6 pt-6">Rank</th>
+        <th class="pb-3 pt-6">Name</th>
+        <th class="pb-3 pt-6">Run Type</th>
```

Comment:
Run type doesn't need to be a column because every run belonging to the cat has the same type. If it's a scored cat then there should be a score column, otherwise a time column and that tells you everything. I suppose I should take this up with the designers too.

Reply:
Should I just keep this in the meantime?